django-20-04/files/var/lib/cloud/scripts/per-instance/001_onboot

```diff
@@ -46,11 +53,37 @@ done
 echo "${DJANGO_USER}:${DJANGO_USER_PASSWORD}" | chpasswd -
 
 # Set up postgres user and database
-setuid postgres psql -U postgres -d postgres -c "CREATE USER ${DJANGO_USER} PASSWORD '${DJANGO_POSTGRESS_PASS}';"
-setuid postgres createdb django
+if [ -f "/root/.digitalocean_dbaas_credentials" ];
```

Comment:
not sure if we are doing this for WP as well, but we should probably check to make sure `/root/.digitalocean_dbaas_credentials` also holds the correct protocol?

if someone is able to spin this up with a mysql or mongo instance, this whole thing will crash and burn

Reply:
Added check for protocol and tested it. If protocol != `postgresql` - it will use local database